package-lock.json

```diff
@@ -1,4733 +1,10578 @@
 {
```

Comment:
npm updated lock file according to `.editorconfig`. Should we keep it since new version of npm will do it for every contributor to the project?